src/panel-triggers/components/AlertList/AlertList.tsx

```diff
@@ -97,15 +103,16 @@ export default class AlertList extends PureComponent<AlertListProps, AlertListSt
     return this.props.onProblemAck(problem, data);
   }
 
-  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, sortOption: string) => {
+  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, testTagFilter: boolean, sortOption: string) => {
     const textFilterLowerCase = textFilter.toLowerCase();
     let filteredProblems = this.props.problems.filter((problem: ProblemDTO) => {
       return (
         (problem.comments.toLowerCase().indexOf(textFilterLowerCase) > -1 ||
         problem.description.toLowerCase().indexOf(textFilterLowerCase) > -1) &&
         (priorityFilter === -1 || problem.severity === priorityFilter.toString()) &&
         (categoryFilter === 'all' || problem.opdata === categoryFilter) &&
-        (!maintenanceFilter || (problem.hosts.length > 0 && problem.hosts[0].maintenance_status === '0'))
+        (!maintenanceFilter || problem.hosts.length === 0 || !problem.hosts.some((h) => h.maintenance_status === '0')) &&
```

Comment:
If I ponder it correctly I think here it must be `=== '1'` ie. there must not be any host with maintenance status `1 - maintenance in effect.`

Reply:
In commit 1681811f318573b1f97e0243f0d8e8f244979c11.